### ISSUE TITLE
How to launch application after install?

### ISSUE THREAD

**Issue by douglasjunior** [felixrieseberg/electron-wix-msi#9]
I did not find any option to launch the app after installation.

**Comment by schontz** [comment]
Also interested in creating a Desktop shortcut...

**Comment by markmorris** [comment]
Added option to launch here e6a800f8c3ded51df5bd3cee2f57fa8cf96f7035
Added Desktop Shortcut here be7017c1af303c67482364327a8e66ede6c6768e

**Comment by schontz** [comment]
Wonderful, thanks! Looking at the commits, I couldn't see any config options. Will these become defaults, or README updates are yet to come?

**Comment by markmorris** [comment]
Yeah this my own fork i would be happy to do a pull request if the main developer was around to check the code. I only started working on this today. The desktop shortcut is always created and will keep the name of the exe. To use the launch option you will need to add `extensions: ['WixUtilExtension']` into the options. I will update the README soon

**Comment by schontz** [comment]
Great. I hope this lands.

**Comment by schontz** [comment]
Could things like this be added via the [Template Configuration](https://github.com/felixrieseberg/electron-wix-msi#template-configuration-optional) settings? I want to install additional files at another path, set an uninstall icon, etc. I wish there was a demo of how to use the template options to see what is available.

**Comment by alexthelion** [comment]
Hi,
Is there a way to launch the app after install?

**Comment by iPzard** [comment]
Anyone figure this out yet? Seems like there's still no docs for this atm.
